From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Nicolas Iooss <nicolas.iooss_linux@m4x.org>
Date: Sun, 16 Jul 2023 10:46:12 +0200
Subject: [PATCH] {PRINTF}
 drivers/platform/x86/intel/int3472/clk_and_regulator: silence
 -Wformat-security with "%s"

---
 drivers/platform/x86/intel/int3472/clk_and_regulator.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/platform/x86/intel/int3472/clk_and_regulator.c b/drivers/platform/x86/intel/int3472/clk_and_regulator.c
index 2a12cd26081d..079fbb8fc603 100644
--- a/drivers/platform/x86/intel/int3472/clk_and_regulator.c
+++ b/drivers/platform/x86/intel/int3472/clk_and_regulator.c
@@ -145,7 +145,7 @@ int skl_int3472_register_dsm_clock(struct int3472_discrete_device *int3472)
 		goto out_free_init_name;
 	}
 
-	int3472->clock.cl = clkdev_create(int3472->clock.clk, NULL, int3472->sensor_name);
+	int3472->clock.cl = clkdev_create(int3472->clock.clk, NULL, "%s", int3472->sensor_name);
 	if (!int3472->clock.cl) {
 		ret = -ENOMEM;
 		goto err_unregister_clk;
-- 
